### ISSUE TITLE
Xcode warning because of bundle

### ISSUE THREAD

**Issue by Robuske** [ArtSabintsev/Siren#357]
Xcode is showing a warning because of the _Siren.bundle_ folder.
Probably is because of the fact that SPM 5.3 supports resources, so it's complaining that the resource is not declared, but I haven't read enough about SPM resources so I am not sure how to fix.

```
found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target

/path/to/folder/Xcode/DerivedData/app-name/SourcePackages/checkouts/Siren/Sources/Siren.bundle
```

**Comment by ArtSabintsev** [comment]
Thanks for the heads up. I'll look into it!

**Comment by ArtSabintsev** [comment]
The issue is now addressed in #358. I'll merge it in soon.

**Comment by Robuske** [comment]
Thanks 😄

**Comment by ArtSabintsev** [comment]
Released as 5.7.0. Thanks again!
